docs/user-guide/configuration-files/strongbox-yaml.md

```diff
@@ -78,6 +79,7 @@ The following is an example of a `strongbox.yaml` configuration file:
               type: hosted
               secured: true
               checksumHeadersEnabled: true
+              strictChecksum: true
```

Comment:
Please, change this to `strictChecksumValidation`.

Reply:
Done!